viomi-root.sh

```diff
@@ -113,9 +113,9 @@ function restore_robot_services() {
 
 function install_valetudo() {
   ip=$1
-  wget "https://github.com/Hypfer/Valetudo/releases/download/0.5.3/valetudo" -O valetudo
+  wget "https://github.com/Hypfer/Valetudo/releases/download/2021.02.0/valetudo" -O valetudo
```

Comment:
This version has new installation instructions: https://valetudo.cloud/pages/installation/viomi.html

So you’d need adjustments below.